doc/benchmark/modules/ROOT/pages/report/result-summary.adoc

```diff
@@ -0,0 +1,51 @@
+= Results summary
+:description: There shell script creates a summary JSON which can be used to compare different runs.
+
+{description}
+
+== JSON created for each run
+
+For every run, `kcb.sh` creates a file `results.json` which contains the input parameters
+to Gatling and also an output of the results.
+
+Each result summarizes a run, and can be processed further in a pipeline.
+Each run has a descriptive name and a UUID which identifies the run.
+The file name contains a timestamp and the UUID, so it is simple to put multiple result files in a single folder for further processing.
+
+When the environment variable `SUT_DESCRIBE` is set, `kcb.sh` calls this script to retrieve a JSON description for the system under test, which is added to the file as well.
```

Comment:
do we have to set this as `SUT_DESCRIBE=true` in the bash session ? similar to the `GRAPHITE_TCP_ADDR` ? @ahus1

Reply:
Yes, it needs to be set in the bash session. You would set it to a full path with CLI parameters to the script that should be called. It could be environment specific. 

It is set here, see the change in this PR: `.github/actions/oc-keycloak-login/action.yml`

Reply:
is it possible to expand on this in the docs ? when you say "`kcb.sh` calls this script" did you mean to have a hyperlink to the rosa-describe.sh script ?

Reply:
Extended the docs.